web/templates/group_invitation/index.html.eex

```diff
@@ -0,0 +1,41 @@
+<h2>Listing Group Invitations</h2>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>User</th>
+      <th>Type</th>
+      <th>Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    <%= for group_membership_request <- @group_membership_requests do %>
+      <tr>
+        <td>
+          <%= group_membership_request.user.name %>
+        </td>
+        <%= if group_membership_request.initiated_by_user do %>
+          <td>User Requested</td>
+          <td>
+            <%= render "accept_invitation_form.html",
+                  conn: @conn,
+                  action: group_invitation_path(@conn, :update, group_membership_request.group_id, group_membership_request) %>
+          </td>
+        <%= else %>
+          <td>Invited by Group</td>
```

Comment:
Are you saying that we don't need to show users this column? I'm a bit confused by "checkbox".